### PR TITLE
Jpk/key serials 5

### PR DESCRIPTION
**Pull request by jpknoll**

### CODE REVIEW

**Comment by jpknoll** [comment]
Fixes #277

File: Keas.Mvc/Controllers/Api/KeySerialsController.cs [hunk 1]
```diff
@@ -107,15 +107,17 @@ public async Task<IActionResult> Create([FromBody] CreateKeySerialViewModel mode
...
+                Number   = model.Number,
             };
 
             // add key serial
             key.Serials.Add(keySerial);
             await _context.SaveChangesAsync();
-
-            //await _eventService.TrackCreateKeySerial(key);
+            
+            //await _eventService.TrackCreateKeySerial(keySerial);
```
Comment (jSylvestre) [inline Keas.Mvc/Controllers/Api/KeySerialsController.cs]: @jcubbage @jpknoll We are not tracking this?

**Comment by jSylvestre** [comment]
Revoke Key Serial is failing.

File: Keas.Mvc/Controllers/Api/KeySerialsController.cs (hunk 1 above)
Comment (jcubbage) [reply Keas.Mvc/Controllers/Api/KeySerialsController.cs]: No...we are tracking keys and assignments. Not key serials.

**Comment by jcubbage** [comment]
> Revoke Key Serial is failing.

Try pulling in master? The history service has been updated since this branch was created.

**Comment by jSylvestre** [comment]
> Fixes #277

FYI, it fixes it for the code, but you can still do it for the name field

**Comment by jpknoll** [comment]
Fixes #297

**Comment by jpknoll** [comment]
Fixes #288

**Comment by jpknoll** [comment]
Fixes #287

**Pull request merged**